.release-it.cjs

```diff
@@ -14,16 +14,19 @@ module.exports = {
 			"sed -i 's/main\\/AUTHORS.txt/${version}\\/AUTHORS.txt/' package.json",
 		"after:bump": "cross-env VERSION=${version} npm run build:all",
 		"before:git:release": "git add -f dist/ dist-module/ changelog.md",
-		"after:release": `bash ./build/release/post-release.sh \${version} ${ blogURL }`
+		"after:release": "echo 'Run the following to complete the release:' && " +
+			`echo './build/release/post-release.sh $\{version} ${ blogURL }'`
 	},
 	git: {
-		changelog: "npm run release:changelog -- ${from} ${to}",
+		changelog: "node build/release/changelog.js ${from} ${to}",
```

Comment:
What's the reason for this change?

Reply:
I tried to describe this in my comment, but I worried that wasn't clear enough. Essentially, when you run an npm script, it first logs the command. The problem with that in this case is that *everything* in stdout is used in the GH release notes. Calling the node script directly fixes that.

Reply:
Ah, I see. A code comment in one of these places with this note could be useful.